guvnor-ala/guvnor-ala-ui/guvnor-ala-ui-client/src/main/java/org/guvnor/ala/ui/client/wizard/source/SourceConfigurationParamsPresenter.java

```diff
@@ -273,30 +273,30 @@ private void loadBranches(final String repository) {
         sourceService.call((Collection<String> branches) -> {
                                view.clearBranches();
                                branches.forEach(view::addBranch);
-                               clearProjects();
+                               clearModules();
                            },
                            new DefaultErrorCallback()
         ).getBranches(repository);
     }
 
     private void loadProjects(String repository,
                               String branch) {
-        sourceService.call((Collection<Project> projects) -> {
-                               clearProjects();
-                               projects.forEach(project -> {
-                                   view.addProject(project.getProjectName());
-                                   currentProjects.put(project.getProjectName(),
-                                                       project);
+        sourceService.call((Collection<Module> modules) -> {
+                               clearModules();
+                               modules.forEach(module -> {
+                                   view.addProject(module.getModuleName());
```

Comment:
Maybe unify the naming by renaming `addProject` to `addModule`?

Reply:
Well spotted. Will rename.